### ISSUE TITLE
StonkOverflow does not support mobile navigation

### ISSUE THREAD

**Issue by qexat** [Yawan-1/StackOverFlow--Clone#1]
That's a really cool project, but it is really sad that it is not fully responsive 😔

**Comment by Yawan-1** [comment]
Yes @qexat , I think that is also a big issue for mobile users, But I have full intentions to make it mobile friendly in few weeks, I would take some time But I will definitely do it. Thanks for Pointing that Out.
